specification/columns/servicecategory.md

```diff
@@ -1,8 +1,8 @@
 # Service Category
 
-The Service Category is the highest-level classification of a service based on the core function of the service. Each service should have one and only one category that best aligns to its primary purpose. The Service Category is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.
+The Service Category is the highest-level classification of a [*service*](#glossary:service) based on the core function of the *service*. Each *service* should have one and only one category that best aligns to its primary purpose. The Service Category is commonly used for scenarios like analyzing costs across providers and tracking the migration of workloads across fundamentally different architectures.
```

Comment:
Can we include this PR as well - https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/pull/251?

Reply:
I think this is resolved. Closing. 

@yprikhodko responded to you on slack. Maintainers decided #251 would be better added post preview release (before v1.0) after feedback. Appreciate you bringing it up.